### ISSUE TITLE
[BUG]Leo 5.9-b2 Open myLeoSettings.leo not work 

### ISSUE THREAD

**Issue by ZoomQuiet** [leo-editor/leo-editor#1260]
env:

    Leo 5.9-b2, build 20190409061733, Tue Apr  9 06:17:32 UTC 2019
    Not running from a git repo
    Python 3.7.3, PyQt version 5.12.3
    darwin
    isPython3: True

`Help -> Open myLeoSettings.leo`

err.:

    no command for @item 'open-leoSettings-leo'
    no command for @item 'open-myLeoSettings-leo'

PS:
my Leo config:

https://github.com/zoom-quiet/scm/blob/master/leo/myLeoSettings.leo

**Comment by technatica** [comment]
Fixed at c5dc9d8
<https://github.com/leo-editor/leo-editor/commit/c5dc9d8f44e7559c917a98fb65e22c483c21dd4d>
in devel.

See #1239

Chris

**Comment by ZoomQuiet** [comment]
@technatica sayeahooo ;-)

thanx, i upgrade into the HEAD, try it.

**Comment by ZoomQuiet** [comment]
![ScreenShot 2019-07-23 09 42 48](https://user-images.githubusercontent.com/22494/61676385-4341d700-ad2e-11e9-9ef3-c8783088cf99.jpg)

@technatica sayeahoo ;-(

look like not merge into master now...

thanx for all,

i waiting,,,

**Comment by technatica** [comment]
It was fixed in devel. Zoom is using an older version from Master.

Chris

**Comment by ZoomQuiet** [comment]
@technatica  thanx for suggest:

checkout devel

   ༼conda: leo373༽⎇ devel ✓༽/opt/bin/leo-editor༽
   ༄  git st
   On branch devel
   Your branch is up-to-date with 'origin/devel'.

   nothing to commit, working tree clean

![Uploading ScreenShot 2019-07-23 23.49.09.jpg…]()



```
   *   fdfa35fdf - (HEAD -> devel, origin/devel) Merge branch 'devel' of https://github.com/leo-editor/leo-editor into devel
   |\      Edward K. Ream 27 hours ago
   | * 74cbfdeb6 - travis: pre-emptively install setupext-janitor
   | |     maphew 5 days ago
   | * 815f62d32 - doc: update pip Requirements; remove py2.7 install notes
   | |     maphew 5 days ago
   | * dda76021e - remove bundled setupext-janitor, closes #1255
   | |     maphew 5 days ago
   | *   c81fa9b10 - Merge branch 'desktop-links' into devel, #1244
   | |\    maphew 7 days ago
   | * | 98b0f97ae - Update branches.md
   | | |   Chris George 9 days ago
   * | |   16374b821 - (origin/desktop-links) Merge branch 'devel' into desktop-links
   |\ \ \          Edward K. Ream 27 hours ago
   | |/ /
   | | /
   | |/
   |/|
   | * 400145176 - Fix #1254

```



and try again:

   ༼conda: leo373༽⎇ devel ✓༽/opt/bin/leo-editor༽
   ༄  python launchLeo.py



but in Leo, also can not open myLeoSettings.leo


```
   Leo Log Window
   Leo 6.0-b2-devel, devel branch, build fdfa35fdf1
   2019-07-22 08:16:42 -0500
   Python 3.7.3, PyQt version 5.12.3
   darwin
   setting leoID from os.getenv('USER'): 'zoomq'
   current dir: /opt/bin/leo-editor
   load dir: /opt/bin/leo-editor/leo/core
   global config dir: /opt/bin/leo-editor/leo/config
   home dir: /Users/zoomq
   reading settings in /opt/bin/leo-editor/leo/config/leoSettings.leo
   reading settings in /Users/zoomq/.leo/myLeoSettings.leo
   duplicate, (not conflicting) key bindings in myLeoSettings.leo
      all Alt+) move-past-close
      all Alt+) move-past-close
   duplicate, (not conflicting) key bindings in myLeoSettings.leo
      all Alt+} forward-paragraph
      all Alt+} forward-paragraph-extend-selection
   duplicate, (not conflicting) key bindings in myLeoSettings.leo
      all Ctrl+( add-comments
      all Ctrl+( add-comments
   duplicate, (not conflicting) key bindings in myLeoSettings.leo
      all Ctrl+) delete-comments
      all Ctrl+) delete-comments
      all Ctrl+) delete-comments
   duplicate, (not conflicting) key bindings in myLeoSettings.leo
      all Ctrl+{ promote
      all Ctrl+{ promote
   duplicate, (not conflicting) key bindings in myLeoSettings.leo
      all Ctrl+} demote
      all Ctrl+} demote
   reading settings in /Users/zoomq/.leo/workbook.leo
   read outline in 0.15 seconds

   no command for @item 'open-myLeoSettings-leo'
   no command for @item 'open-leoSettings-leo'

```

**Comment by edreamleo** [comment]
The `@item` names are:
```
@item open-my-leo-settings
@item open-leo-settings
```
These may have changed recently.  Please change them in your myLeoSettings.leo file.

Edward

**Comment by ZoomQuiet** [comment]
@edreamleo thanx for suggest,
but: try fixed in myLeoSettings.leo as

![ScreenShot 2019-07-24 23 56 38](https://user-images.githubusercontent.com/22494/61808828-c5391980-ae6e-11e9-8e21-422fa24297ee.jpg)

but the menu not changed:

![ScreenShot 2019-07-24 23 55 05](https://user-images.githubusercontent.com/22494/61808820-bf433880-ae6e-11e9-8531-de3a4e2314db.jpg)

same err.:

...

no command for @item 'open-leoSettings-leo'
no command for @item 'open-myLeoSettings-leo'


## guess 
myLeoSettings.leo is usage since Leo 3.5 ...

maybe include so many old items,

so my err. is replace as:
How re-config myLeoSettings.leo base lasted new start myLeoSettings.leo?

**Comment by technatica** [comment]
The menu was updated in the fix for #1239 so you would need to either grab
the @menu from LeoSettings.leo and bring them into your myLeoSettings.leo
or you could just change

@item <https://github.com/item> 'open-leoSettings-leo'
@item <https://github.com/item> 'open-myLeoSettings-leo'

to

@item <https://github.com/item> 'open-leoSettings'
@item <https://github.com/item> 'open-myLeoSettings'

in your myLeoSettings.leo.

HTH,

Chris

**Comment by maphew** [comment]
@technatica Github did something weird with the formatting of your message, inserting links. Perhaps edit the comment in-situ to fix (https://github.com/leo-editor/leo-editor/issues/1260#issuecomment-514699543)
